modules/cdf_infield_location/default.config.yaml

```diff
@@ -21,7 +21,7 @@ infield_location_viewer_users_source_id: 7bdcb20c-3e6a-400c-b9ef-cf835f64f05e
 # Transformation credentials
 clientId: ${IDP_CLIENT_ID}
 clientSecret: ${IDP_CLIENT_SECRET}
-tokenUrl: ${IDP_TOKEN_URL}
+tokenUri: ${IDP_TOKEN_URL}
```

Comment:
Why do you want to switch to tokenUri? It will always be a url.

Reply:
It is actually not, transformations has `tokenUri`, see the current version of the SDK https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/data_classes/transformations/common.py#L300

As well as the spec:
![image](https://github.com/cognitedata/cdf-project-templates/assets/60234212/0dbe5a81-fdee-4d38-9e77-60be86de6001)

Reply:
Hm, this is bad as we always refer to tokenUrl other places (which is more specific). Although URI is not wrong per se, it is imprecise as it always will be a url. Ok, I guess we should let the API spec trump consistency here?

Reply:
I guess so, unfortunately